### PR TITLE
Fix control disable

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-kdump.changes [hunk 1]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Dec 15 11:34:57 UTC 2017 - jreidinger@suse.com
+
+- Show kdump as disabled in proposal if product disable it
+  (bsc#1071896)
```
Comment (mvidner) [inline package/yast2-kdump.changes]: "selecting a volume group on a disk without partitions causes ...",are you sure it's the right bug?

File: package/yast2-kdump.changes [hunk 2]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Dec 15 11:34:57 UTC 2017 - jreidinger@suse.com
+
+- Show kdump as disabled in proposal if product disable it
```
Comment (mvidner) [inline package/yast2-kdump.changes]:
```
Thank you for using GrammarCop!
2    MissingArticles/The          (Warning)
1    VerbForms/3rdPersonSingular  (Error)
--
3    Total
```

File: package/yast2-kdump.changes (hunk 1 above)
Comment (jreidinger) [reply package/yast2-kdump.changes]: Ah, my fault. Too much open bugs. Will fix it.

File: package/yast2-kdump.changes [hunk 3]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Dec 15 11:34:57 UTC 2017 - jreidinger@suse.com
+
+- Show kdump as disabled in an installation proposal if the product
```
Comment (mvidner) [inline package/yast2-kdump.changes]:
```
Thank you for using GrammarCop!

1    IncorrectArticles/UniqueReferent (Warning)
--
1    Total
```

**Pull request merged**